core/locale/main/values-ar/strings_ui.xml

```diff
@@ -6,7 +6,7 @@
     <string name="content_type_multimedia">وسائط متعددة (صوت و صورة)</string>
     <string name="multiple_choice">متعدد الخيارات</string>
     <string name="content_type_fill_the_blanks">إملأ الفراغات</string>
-    <string name="content_type_link">ربط</string>
+    <string name="link">ربط</string>
```

Comment:
We must not change the translated xmls. These are generated by Weblate. Changing them might introduce a conflict. Can you please revert all non-english string xml files? Keep the change in English. Weblate will figure it out from there.